core/src/main/java/com/datastax/oss/driver/internal/core/metadata/DefaultTopologyMonitor.java

```diff
@@ -89,7 +89,9 @@ public DefaultTopologyMonitor(InternalDriverContext context) {
     if (closeFuture.isDone()) {
       return CompletableFutures.failedFuture(new IllegalStateException("closed"));
     }
-    return controlConnection.init(true, reconnectOnInit, true);
+    // we only want to us the initial reconnection schedule here if reconnectOnInit is true
+    final boolean useInitialReconnectionSchedule = reconnectOnInit;
+    return controlConnection.init(true, reconnectOnInit, useInitialReconnectionSchedule);
```

Comment:
We do not need this change if we should always set `useInitialReconnectionSchedule` to true and we simply log a warning when `reconnectOnInit` is `false`. However, I'm not certain on if/when we should set `useInitialReconnectionSchedule` to `true` here. Perhaps there is more logic that needs to be evaluated?

Reply:
After discussing this more, we're simply going to remove the Precondition, as it's not an abnormal situation (as the javadocs suggest).